src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -229,6 +229,9 @@ def load_arguments(self, _):
         with self.argument_context(scope + ' config ssl import') as c:
             c.argument('key_vault', help='The name or resource ID of the Key Vault')
             c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault')
```

Comment:
Shouldn't this be marked as deprecated in future releases? Moving forward only single parameter should be supported.